submissions/Ant-C-tech/friends-app/js/app.js

```diff
@@ -0,0 +1,314 @@
+'use strict'
+
+const MAIN = document.querySelector('#main')
+const MUSIC_BUTTON = document.querySelector('#musicButton')
+const SEARCH = document.querySelector('.search')
+const SEARCH_INPUT = document.querySelector('#icon_prefix')
+const RESET_BUTTON = document.querySelector('.resetButton')
+
+const API_LINK = 'https://randomuser.me/api/'
+
+const FRIENDS_MIN = 30
+const FRIENDS_MAX = 50
+
+const APP_AUDIO = new Audio('./audio/you_ve_got_a_friend_in_me.mp3')
+APP_AUDIO.loop = true
+APP_AUDIO.volume = 0.3
+
+//Classes by AnimateCSS (default speed is 1s)
+const SHOW_ELEM_PRIMARY_ANIMATION = 'animate__zoomIn'
+const HIDE_ELEM_PRIMARY_ANIMATION = 'animate__zoomOut'
+const SHOW_ELEM_SECONDARY_ANIMATION = 'animate__fadeIn'
+const HIDE_ELEM_SECONDARY_ANIMATION = 'animate__fadeOut'
+const ANIMATION_SPEED = 'animate__faster' //500ms
+
+let FRIENDS_SOURCE
+let CURRENT_FRIENDS = []
+let IS_MUSIC_STOPPED_BY_USER = false
+let CURRENT_CONTENT
+let CURRENT_ANIMATION_HIDE
+let CURRENT_ANIMATION_SHOW
+let CURRENT_ANIMATION_SPEED
+
+
+initApp()
+
+
+function initApp() {
+    getFriends(getRandomIntInclusive(FRIENDS_MIN, FRIENDS_MAX))
```

Comment:
Loading of a random number of users is a bad idea. Imagine such logic in FB or Twitter :)

Reply:
What the motivation behind this idea?

Reply:
This app for me looks not like Twitter or FB. There is no any posibility to interact with people. Maybe it looks like tiny app in which stars or maybe octopus Paul gets you some list of people who you can call or something. I do not know why Paul choose these people, so I do not know how many people it can be. That is why I chosen random quantity and limit it to have more reality.
Anyway, I removed this code.